tests/acceptance/acceptancesuites/filterboxsuite.js

```diff
@@ -84,23 +84,21 @@ test(`multioption filterbox works with back/forward navigation and page refresh`
   await searchComponent.submitQuery();
 
   // Click the 'Marty' checkbox
-  // 2 filter tags should be rendered, 1 from the backend facet and 1 from the static filter
   await staticFilter.toggleOption('Marty');
   await expectRequestFiltersToEql(filterBoxLogger, martyFilter);
-  await t.expect(filterTags.count).eql(2);
+  await t.expect(filterTags.count).eql(1);
```

Comment:
for my own edification, why is this changing?

Reply:
yup! took the facets off the make the tests a little more focused